bug:Local Notifications--- Notifications not enabled on this device

# Bug Report

## Capacitor Version
<!-- Provide the version of Capacitor and related installed dependencies. You can use the command `npx capacitor doctor` or `npx cap doctor` for the output from the root directory of your project. -->
Capacitor Version:2.0 And 1.5 are same.
`npx cap doctor` output:
NO Problem

## Affected Platform(s)
<!-- Leave these checkboxes EMPTY until after the issue is created. Once the issue is created, check one or more boxes for the platform(s) this bug affects. -->

- [1 ] Android
- [ ] iOS
- [ ] Electron
- [ ] Web

## Current Behavior
V/Capacitor/Plugin: To native (Capacitor plugin): callbackId: 111430489, pluginId: LocalNotifications, methodName: areEnabled
V/Capacitor: callback: 111430489, pluginId: LocalNotifications, methodName: areEnabled, methodData: {}
V/Capacitor/Plugin: To native (Capacitor plugin): callbackId: 111430490, pluginId: LocalNotifications, methodName: schedule
V/Capacitor: callback: 111430490, pluginId: LocalNotifications, methodName: schedule, methodData: {"notifications":[{"body":"This is a fine SkekNot","id":1,"title":"Skekification!","sound":null,"schedule":{"repeats":true,"every":"second","count":10}}]}
D/Capacitor: Sending plugin error: {"save":false,"callbackId":"111430490","pluginId":"LocalNotifications","methodName":"schedule","success":false,"error":{"message":"Notifications not enabled on this device"}}
E/Capacitor: Serious error executing plugin
    java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:99)
        at com.getcapacitor.Bridge$1.run(Bridge.java:520)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.os.HandlerThread.run(HandlerThread.java:61)
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int org.json.JSONArray.length()' on a null object reference
        at com.getcapacitor.plugin.LocalNotifications.schedule(LocalNotifications.java:79)
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:99) 
        at com.getcapacitor.Bridge$1.run(Bridge.java:520) 
        at android.os.Handler.handleCallback(Handler.java:751) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.os.HandlerThread.run(HandlerThread.java:61) 
E/AndroidRuntime: FATAL EXCEPTION: CapacitorPlugins
    Process: com.example.app, PID: 1099
    java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
        at com.getcapacitor.Bridge$1.run(Bridge.java:529)
        at android.os.Handler.handleCallback(Handler.java:751)
        at android.os.Handler.dispatchMessage(Handler.java:95)
        at android.os.Looper.loop(Looper.java:154)
        at android.os.HandlerThread.run(HandlerThread.java:61)
     Caused by: java.lang.reflect.InvocationTargetException
        at java.lang.reflect.Method.invoke(Native Method)
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:99)
        at com.getcapacitor.Bridge$1.run(Bridge.java:520)
        at android.os.Handler.handleCallback(Handler.java:751) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.os.HandlerThread.run(HandlerThread.java:61) 
     Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int org.json.JSONArray.length()' on a null object reference
        at com.getcapacitor.plugin.LocalNotifications.schedule(LocalNotifications.java:79)
        at java.lang.reflect.Method.invoke(Native Method) 
        at com.getcapacitor.PluginHandle.invoke(PluginHandle.java:99) 
        at com.getcapacitor.Bridge$1.run(Bridge.java:520) 
        at android.os.Handler.handleCallback(Handler.java:751) 
        at android.os.Handler.dispatchMessage(Handler.java:95) 
        at android.os.Looper.loop(Looper.java:154) 
        at android.os.HandlerThread.run(HandlerThread.java:61) 
E/Capacitor/Console: File: http://localhost/main-es5.6ea357cb770a27bd4e8a.js - Line 1 - Msg: ERROR
I/Process: Sending signal. PID: 1099 SIG: 9

--- a/android/capacitor/src/main/java/com/getcapacitor/plugin/LocalNotifications.java
+++ b/android/capacitor/src/main/java/com/getcapacitor/plugin/LocalNotifications.java
@@ -73,18 +73,20 @@ public class LocalNotifications extends Plugin {
       return;
     }
     JSONArray ids = manager.schedule(call, localNotifications);
-    notificationStorage.appendNotificationIds(localNotifications);
-    JSObject result = new JSObject();
-    JSArray jsArray = new JSArray();
-    for (int i=0; i < ids.length(); i++) {
-      try {
-        JSObject notification = new JSObject().put("id", ids.getString(i));
-        jsArray.put(notification);
-      } catch (Exception ex) {
+    if (ids != null) {
+      notificationStorage.appendNotificationIds(localNotifications);
+      JSObject result = new JSObject();
+      JSArray jsArray = new JSArray();
+      for (int i=0; i < ids.length(); i++) {
+        try {
+          JSObject notification = new JSObject().put("id", ids.getString(i));
+          jsArray.put(notification);
+        } catch (Exception ex) {
+        }
       }
+      result.put("notifications", jsArray);
+      call.success(result);
     }
-    result.put("notifications", jsArray);
-    call.success(result);
   }
 
   @PluginMethod()

